src/pages/index.js

```diff
@@ -124,4 +125,93 @@ const enableValidation = (config) => {
   });
 };
 
+//initializations
+
+const user = new UserInfo(userData);
+
+const profilePopup = new PopupWithForm(editProfilePopup, {
+  handleFormSubmit: ({ name, about_me }) => {
+    const profileInfoSubmit = document.querySelector("#profile-info-submit");
+    profileInfoSubmit.textContent = "Saving...";
+    user.setUserInfo({ newName: name, newJob: about_me });
+    api.patchInfo({name, about: about_me});
+    setTimeout(profilePopup.close(), 300); //I am aware this isn't necessarily the same thing as having it close when the api is finished- this is a simulacrum of it. I couldn't figure out how to connect it all. You will see this replicated elsewhere
+  },
+});
+
+const addPopup = new PopupWithForm(addPhotoPopup, {
+  handleFormSubmit: ({ title, image_url: imageUrl }) => {
+    const newObj = {
+      name: title,
+      link: imageUrl
+    };
+    api.postCard(newObj);
+    cardSection.clear();
```

Comment:
`cardSection.clear();  ` and `loadApiCards();` are not needed

* When I add new cards, everything disappers from the page. It should not happen because it's a huge waste of resources on refetching and rerendering
* The same is true for deleting cards